Commands/Commands.cs

```diff
@@ -23,5 +23,14 @@ public async Task GetLatency(CommandContext ctx)
                 .Build();
             await ctx.Channel.SendMessageAsync(embed: latency);
         }
+        [Command("F")]
```

Comment:
So there's a problem there, we don't only want the command "F", we also want some limitations i.e,
- Cooldown of 3 seconds
- A summary/description

Reply:
Ok